### PR TITLE
Add teb local planner

### PR DESCRIPTION
**Pull request by serraramiro1**
Adds teb local planner, a planner that finds the path that will take the least to be executed.
This means, going backwards, turning while moving, etc.
Quick showcase:
teb planner

Setting LOCAL_PLANNER=teb env variable will enable the planner.

The gif was recorded with:
`LOCAL_PLANNER=teb RVIZ_CONFIG=navigation LOCALIZATION=amcl roslaunch ca_gazebo create_house.launch`

### CODE REVIEW

File: docker/create_ros_melodic_gazebo9/packages.txt [hunk 1]
```diff
@@ -43,3 +43,4 @@ ros-*-smach-viewer
 ros-*-transmission-interface
 ros-*-xacro
 ros-*-yocs-controllers
+ros-*-teb-local-planner
```
Comment (eborghi10) [inline docker/create_ros_melodic_gazebo9/packages.txt]: Add it alphabetically, please.

File: navigation/ca_move_base/config/teb_local_planner.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
+base_local_planner: "teb_local_planner/TebLocalPlannerROS"
+
+TebLocalPlannerROS:
```
Comment (eborghi10) [inline navigation/ca_move_base/config/teb_local_planner.yaml]: Would you add a link to the reference? And also some useful comments that you consider, this means, comments that are in the ROS Wiki for particular parameters or personal comments based on your experience.

File: navigation/ca_move_base/config/teb_local_planner_params.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
+base_local_planner: "teb_local_planner/TebLocalPlannerROS"
+
+TebLocalPlannerROS:
```
Comment (eborghi10) [inline navigation/ca_move_base/config/teb_local_planner_params.yaml:4]: What's the difference between this and the other file?

**Comment by eborghi10** [comment]
[Travis build](https://travis-ci.org/github/RoboticaUtnFrba/create_autonomy/builds/688172556) passing so far :+1: 

BTW, I don't know why Travis stopped reporting the error,

**Comment by eborghi10** [comment]
One more comment, you need to add TEB as dependency in [`ca_move_base` package](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/navigation/ca_move_base/package.xml).

**Comment by eborghi10** [comment]
Running instructions?

**Review by eborghi10 — COMMENTED** [review]
> Running instructions?

I used this command and it's working _REALLY_ **well** even with closed turnings. Congrats!!

```bash
 LOCALIZATION=amcl RVIZ_CONFIG=navigation LOCAL_PLANNER=teb roslaunch ca_gazebo create_house.launch
```

Did you tweak something more?

File: navigation/ca_move_base/config/teb_local_planner_params.yaml (hunk 3 above)
Comment (serraramiro1) [reply navigation/ca_move_base/config/teb_local_planner_params.yaml:4]: The other file was redundant. deleted

File: navigation/ca_move_base/config/teb_local_planner.yaml (hunk 2 above)
Comment (serraramiro1) [reply navigation/ca_move_base/config/teb_local_planner.yaml]: Added link to reference, and some comments on some parameters

File: docker/create_ros_melodic_gazebo9/packages.txt (hunk 1 above)
Comment (serraramiro1) [reply docker/create_ros_melodic_gazebo9/packages.txt]: Addressed

**Review by eborghi10 — APPROVED** [review]

**Pull request merged**